packages/generate-arg-types/src/arg-types.ts

```diff
@@ -34,67 +35,64 @@ export const propsToArgTypes = (
       result[propName] = argType;
     }
     return result;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  }, {} as Record<string, any>);
+  }, {} as Record<string, PropMeta>);
 };
 
 const matchers = {
   color: new RegExp("(background|color)", "i"),
   date: /Date$/,
 };
 
+const toPropMeta = (
+  controlType: PropMeta["controlType"],
+  dataType: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
do we really have to use any here? in such cases we need to comment the "why"

Reply:
maybe something like unknown works?

Reply:
This is existing code, I didn't add this. Can spend some time cleaning up types in this file if you want

Reply:
try for a few minutes, if it gets weird we can ask a Andarist, it may be a long tail that has resulted in it